### ISSUE TITLE
Will this programme work with Serial Cable Communication? Can you submit a example of executing this programme?

### ISSUE THREAD

**Issue by sameer546** [rimek/fatek-fbs-lib#15]

**Comment by rimek** [comment]
No, it was designed to be used with tcp communication over ethernet:
https://github.com/rimek/fatek-fbs-lib/blob/master/fatek/fatek.py#L21
You would have to change this to `ModbusSerialClient`, then it might work

**Comment by sameer546** [comment]
I tried using the ModbusSerialClient, But it was not working.
Do I need to configure the Modbus Settings inside the PLC for this communication.
Can't I directly tap the data from the PLC..(like X0, X1, Y1, C1 etc.) without tampering the plc settings???
